ecs-cli/modules/cli/compose/project/project_parseV3_test.go

```diff
@@ -14,45 +22,88 @@ func TestParseV3WithOneFile(t *testing.T) {
 	composeFileString := `version: '3'
 services:
   wordpress:
+    cap_add:
+      - ALL
+    cap_drop:
+      - NET_ADMIN
+    command: echo "hello world"
     image: wordpress
+    entrypoint: /wordpress/entry
     ports: ["80:80"]
+    dns:
+      - 2.2.2.2
+    dns_search:
+      - wrd.search.com
+      - wrd.search2.com
+    labels:
+      com.example.wordpress: "wordpress label"
+    links:
+      - mysql
+    logging:
+      driver: awslogs
+      options:
+        awslogs-group: mywrdprs-logs
+        awslogs-region: us-west-2
+        awslogs-stream-prefix: wordpress
+    tmpfs:
+      - "/run:rw,size=1gb"
+    read_only: true
   mysql:
-    image: mysql`
+    image: mysql
+    labels:
+      - "com.example.mysql=mysqllabel"
+      - "com.example.mysql2=anothermysql label"
+    user: mysqluser
+    hostname: mysqlhost
+    security_opt:
+      - label:role:ROLE
+      - label:user:USER
+    working_dir: /mysqltestdir
+    privileged: true
+    extra_hosts:
+      - "mysqlexhost:10.0.0.0"`
+	logrus.Info("compose string: %s", composeFileString)
 
 	tmpfile, err := ioutil.TempFile("", "test")
 	if err != nil {
 		t.Fatal("Unexpected error in creating test file: ", err)
 	}
 	defer os.Remove(tmpfile.Name())
 
-	if _, err := tmpfile.Write([]byte(composeFileString)); err != nil {
-		t.Fatal("Unexpected error writing to test file: ", err)
-	}
-	if err := tmpfile.Close(); err != nil {
-		t.Fatal("Unexpected error closing test file: ", err)
-	}
+	_, err = tmpfile.Write([]byte(composeFileString))
+	assert.NoError(t, err, "Unexpected error parsing file")
+
+	err = tmpfile.Close()
+	assert.NoError(t, err, "Unexpected error parsing file")
 
 	// add files to projects
 	project := setupTestProject(t)
 	project.ecsContext.ComposeFiles = append(project.ecsContext.ComposeFiles, tmpfile.Name())
 
-	// assert number of container configs matches expected number of services
-	// TODO: assert containerConfig content after conversion implemented
+	// get map of docker-parsed ServiceConfigs
+	expectedConfigs := getServiceConfigMap(t, project.ecsContext.ComposeFiles)
+
+	// assert # and content of container configs matches expected services
 	conConfigResult, err := project.parseV3()
```

Comment:
Nit: since you have an `expectedConfigs`, i usually like having this be `actualConfigs`.  Seems a teeny bit less clunky than having something with `conConfig` in the name...